Included team growth tracking practices (#26)
CCI-MOC/ops-docs: practices/0005-team-growth-tracking.md

# Team Growth Tracking

We believe strongly in the growth of our team members. Expanding our
knowledge and cohesiveness is a constant aspiration. With that goal in mind,
we have created a GitHub Project and Epics to help track our effort in
expanding our team skills both technical and personal. We have also created
GitHub issue [templates][1] to speed up the process of submitting a team
growth issue.

[1]:https://github.com/CCI-MOC/ops-private/issues/templates/edit

> 

Reviewer: Just a comment; not a request for a change:

For a document like this, it often makes more sense to use named rather than numbered links. That is, rather than:

```
GitHub issue [templates][1] to speed up the process of submitting a team

[1]: https://github.com/CCI-MOC/ops-private/issues/templates/edit
```

You can do this if the link name is the same as the link title:

```
GitHub issue [templates][] to speed up the process of submitting a team

[templates]: https://github.com/CCI-MOC/ops-private/issues/templates/edit
```

Or you can do something like this:

```
GitHub issue [templates][foo] to speed up the process of submitting a team

[foo]: https://github.com/CCI-MOC/ops-private/issues/templates/edit
```

This makes life easier as you add or re-arrange links in the document.

in mind when creating/updating your Team Growth Issues.

1. Break your issues down into pieces that can be finished in 1 sprint.

1. Including after-thoughts and ratings for the training that you take will
not only help you in the future but also others that are interested in the
same training.

1. Taking quick notes as you work your way through the training will not
only help you to track where you are but will also allow others to have a
feel for how the training is going and might raise useful informative
questions/conversations.

### Epic - Self-training

Self-training is an important part of growing the team’s skill sets. We can
train ourselves in multiple different ways.

1. We can create projects for ourselves based on previous Organized Training.

1. We can find self-guided training online.

1. We can read a book written about the skill set we are growing.

### Epic - Organized Training

Trainings organized by BU, RedHat or one of our other alliance members would
be considered Organized Training. Some examples of Organized training would
be:

1. Affiliated University courses on technology, leadership,
management, or communication.

>    1. BU [Tarrier eDevelopment][2]

Reviewer: Replace "Tarrier" with "Terrier"

>       1. For example[Essentials for Program Managers][5]

Reviewer: There is a space (and perhaps a comma) missing after "For example".

   1. Courses such as [Cloud Computing (CAS CS 528)][3]

1. RedHat Live or Web-based training.

1. Approved training seminars/lecture series.

1. Approved team cross training.

[2]:http://terrieredev.bu.edu/
[3]:https://www.bu.edu/academics/cas/courses/cas-cs-528/
[5]:https://www.bu.edu/odl/what-we-offer/professional-development-for-managers-and-leaders/program-manager-essentials/

### Epic - Community Building

> Improving the cohesiveness of our team will help in many ways least of which
> is ensuring we communicate effectively when working on projects. To that end,
> we will create the space for our team to work on community-building
> activities. Some examples of this would be:

Reviewer: ```suggestion
Improving the cohesiveness of our team will help us communicate more
effectively when working on projects. To that end, we will create a space for
our team to work on community-building activities. Some examples of this would
be:
```
